packages/webapp/src/components/Expandable/styles.module.scss

```diff
@@ -36,3 +37,15 @@
 .mainContentWrapper.leftCollapse {
   order: 1;
 }
+
+.pill {
+  height: 24px;
+  padding: 8px 16px;
+  background: var(--Colors-Neutral-Neutral-50);
+  color: var(--Colors-Neutral-Neutral-600);
+  width: 100%;
+  justify-content: center;
+  @include lg-breakpoint {
```

Comment:
The mobile layout should extend to 1200px? Or should `@include` here and in .alwaysVisibleContent be sm-breakpoint?

Reply:
Yes because thats what animal inventory is 🤷 .. it needs to match.

Reply:
Okay I'll make sure to leave a note in the bug bash table about the mobile view inventory at 800-1200px; looks totally bonkers to me (bonkers on the headers + padding I mean, not the table rows)